packages/editor/src/lib/editor/managers/LicenseManager.ts

```diff
@@ -1,11 +1,22 @@
 import { T } from '@tldraw/validate'
 import nacl from 'tweetnacl'
 import util from 'tweetnacl-util'
+import { publishDates } from '../../../version'
+
+const GRACE_PERIOD_DAYS = 5
+
+const FLAGS = {
+	annualLicense: 1,
```

Comment:
nit: let's use hex here (it'll be slightly easier to maintain).

e.g.
```

const FLAG_ANNUAL_LICENSE = 0x01
const FLAG_LIFETIME_LICENSE = 0x02
const FLAG_PERPETUAL_LICENSE = 0x04
const FLAG_DOMAIN_LOCKED = 0x08
const FLAG_STUFF = 0x10
const FLAG_OTHER = 0x20
```

Reply:
I don't like the `FLAGS` prefix, feels so repetitive. Having an object also steers people into adding the flags there instead of (potentially) scattering them around in different places. Did change it to hex 👍

Reply:
sounds good!